exercises/allergies/allergies_test.go

```diff
@@ -67,9 +67,9 @@ func TestAllergicTo(t *testing.T) {
 	for _, test := range allergicToTests {
 		actual := AllergicTo(test.i, test.allergen)
 		if actual != test.expected {
-			t.Fatalf("FAIL: AllergicTo(%d, %s): expected %t, actual %t", test.i, test.allergen, test.expected, actual)
+			t.Fatalf("FAIL: AllergicTo(%d, %q): expected %t, actual %t", test.i, test.allergen, test.expected, actual)
 		} else {
-			t.Logf("PASS: AllergicTo(%d, %s) %t", test.i, test.allergen, actual)
+			t.Logf("PASS: AllergicTo(%d, %q) %t", test.i, test.allergen, actual)
```

Comment:
same: this one wouldn't strictly have been necessary since it's just a single string (and we know it will always be non-empty since the test provides it, not the student solution), but won't hurt to have.